discord/http.py

```diff
@@ -420,9 +421,10 @@ def send_message(
         tts: bool = False,
         embed: Optional[embed.Embed] = None,
         embeds: Optional[List[embed.Embed]] = None,
-        nonce: Optional[str] =  None,
-        allowed_mentions: bool = None,
-        message_reference: bool = None,
+        nonce: Optional[str] = None,
+        allowed_mentions: Optional[bool] = None,
+        message_reference: Optional[bool] = None,
+        stickers: Optional[List[sticker.StickerItem]] = None,
```

Comment:
Not familiar with typing. Is there a reason it's typed `Optional` here and not in `abc.Messageable.send`? Noticed many other attributes not typed as `Optional` there as well.